frontend/src/components/RootModal/index.tsx

```diff
@@ -0,0 +1,52 @@
+import Modal from '@material-ui/core/Modal';
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import CreateUserModal from '../CreateUserModal';
+import DeleteUserModal from '../DeleteUserModal';
+import { ModalProps } from '../../model';
+import {
+  hideModal,
+  ModalConfiguarion,
+  ModalType,
```

Comment:
config typo in this file

Reply:
Oops, I will add spell checker extension :facepalm: